extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ * CopySide.RIGHT 2017 The AMP HTML Authors. All Side.RIGHTs Reserved.
```

Comment:
LOL

Reply:
This is what happens when ctrl+f betrays you! (because you are too lazy 😢)

Reply:
> You're doing amazing, sweetie!

-- Kristen Mary Jenner